src/custom/docs/components/footer/index.tsx

```diff
@@ -1,12 +1,56 @@
 import { QuantinuumLogo } from "../../QuantinuumLogo";
 import { Separator } from "src";
-export const Footer = (props: { subtitle: string, columns: {
-    title: string;
-    items: {
-        name: string;
-        href: string;
-    }[];
-}[] }) => {
+
+const footerConfig = {
+  columns: [
+    {
+      name: "Products",
+      items: [
+        {
+          name: "H-Series",
+          href: "https://docs.quantinuum.com/h-series"
```

Comment:
We could probably remove the `https://docs.quantinuum.com` prefix for internal links.